guide/english/jquery/jquery-chaining/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: jQuery Chaining
+---
+## jQuery Chaining
+jQuery chaining allows you to execute multiple methods on the same jQuery selection, all on a single line.
+
+Chaining allows us to turn multi-line statements:
+```javascript
+$('#someElement').removeClass('classA');
+$('#someElement').addClass('classB');
+```
+Into a single statement:
+```javascript
+$('#someElement').removeClass('classA').addClass('classB');
+```
+
+#### More Information:
+For more information, please visit the <a href='https://www.w3schools.com/jquery/jquery_chaining.asp' target='_blank' rel='nofollow'>w3schools documentation</a>.
```

Comment:
Please change this line to the markdown link format

Reply:
Fixed in my recent commit